src/main/java/rocks/inspectit/gepard/config/model/instrumentation/rules/RuleTracingConfiguration.java

```diff
@@ -0,0 +1,23 @@
+/* (C) 2024 */
+package rocks.inspectit.gepard.config.model.instrumentation.rules;
+
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+/** Represents the tracing configuration for a rule. */
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@EqualsAndHashCode
+public class RuleTracingConfiguration {
+
+  /** The configuration representing that no tracing will be done by inspectIT */
+  public static final RuleTracingConfiguration NO_TRACING = new RuleTracingConfiguration();
+
+  /** If true, the methods, which are matched by the rule will appear in traces. */
+  @Valid @NotNull private Boolean startSpan = false;
```

Comment:
Boolean or boolean? 
In ScopeConfiguration.java, you are using boolean.
Try to be consistent here.

Reply:
Changed to `boolean`. Within collection types, you cannot use primitives.